### ISSUE TITLE
pmt init leads to error

### ISSUE THREAD

**Issue by jaaneh** [Errorname/prisma-multi-tenant#72]
Hey there!

I'm trying to set up pmt so I am able to access two databases in the same app through prisma.

After installing prisma-multi-tenant, running `pmt init` and giving it the management db url, it hits an `Cannot read property 'plural' of undefined` at the _Creating example script..._ step.

![](https://jaany.xyz/i/lgg68.png)

I am also getting this warning when going through init `Warning: This version of prisma-multi-tenant is compatible with @prisma/cli@2.4.1, but you have @prisma/cli@2.10.1 installed. This may break in unexpected ways.` 

My local and globally installed @prisma/cli are at v2.4.1, so I'm not sure why this shows up even. Not sure if this could be related to the init error?

I've reinstalled all packages multiple times and I'm really at a loss at what to do here.

```json
// global
"prisma-multi-tenant@2.3.1"
// deps
"@prisma-multi-tenant/client": "^2.3.1",
"@prisma/client": "^2.9.0",
// dev dep
"@prisma/cli": "2.4.1",
```

Any nudge in the right direction is greatly appreciated.

**Comment by Errorname** [comment]
Hi @jaaneh ! Thanks for the issue 🙏  There was indeed a bug following a change in Prisma. This should now have been resolved with the release of prisma-multi-tenant@2.4.1. Don't hesitate to reopen the issue if you still experience the issue

**Comment by jaaneh** [comment]
Hey, thanks!

The current 2.4.1 leads to this, do you happen to know what could be the reason? ![](https://jaany.xyz/i/76590.png)

**Comment by jaaneh** [comment]
@Errorname Hey! I'm stuck at a EISDIR read error after upgrading to 2.4.1. Any ideas? 

![](https://camo.githubusercontent.com/ed90cb86be0d399054f3f61dc3a6ddefa45933db/68747470733a2f2f6a61616e792e78797a2f692f37363539302e706e67)

**Comment by Errorname** [comment]
Ah! Yes, it's an issue with Windows and directories with a space in them. I'll fix this and release a new version

**Comment by jaaneh** [comment]
> Ah! Yes, it's an issue with Windows and directories with a space in them. I'll fix this and release a new version

Appreciate it! I've stumbled across so many errors due to having 2 first names.

**Comment by Errorname** [comment]
It should now be fixed with pmt@2.4.2

**Comment by jaaneh** [comment]
We have a `multi-tenancy-example.js` file! Appreciate the help :-)
